lib/Runtime/PlatformAgnostic/Platform/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(PL_SOURCE_FILES
   Linux/UnicodeText.ICU.cpp
   Linux/NumbersUtility.cpp
   Linux/Thread.cpp
+  Linux/PerfTrace.cpp
```

Comment:
The `perf` tool is, but the idea of a mapping from memory addresses to functions is not, and I don't believe any of the mechanisms I have used are not compatible with OSX.